src/admin/components/views/collections/List/Cell/field-types/Upload/index.scss

```diff
@@ -3,6 +3,6 @@
 .upload-cell {
   &--mime-type {
     color: $color-gray;
-    float: right;
+    float: inline-start;
```

Comment:
I think if we just remove this property altogether, the `span`s will automatically sit next to each other as they should be set to `display: inline` by default

Reply:
Will remove mime type from cell